src/main/java/org/repodriller/scm/GitRepository.java

```diff
@@ -149,7 +143,7 @@ public ChangeSet getHead() {
         try (Git git = openRepository()) {
 			ObjectId head = git.getRepository().resolve(Constants.HEAD);
 
-			RevWalk revWalk = new RevWalk(git.getRepository());
+			RevWalk revWalk = new RevWalk(git.getRepository()); /* TODO Leak. Use try-with. */
```

Comment:
`getRepository()` needs to be closed? Are you sure that closing `Git git` is not enough? I guess that's what we do in this entire class.

Reply:
My IDE says it should be closed. I didn't investigate, but I think the IDE was wrong. I'll remove this.

Reply:
Also just checked the source code, and if you close the `Git`, it closes its repo. You can safely remove it.

Reply:
Will do.